app/gui/CHANGELOG.md

```diff
@@ -6,10 +6,14 @@
 - [Fixed broken node whose expression contains non-ASCII characters.][3166]
 - [Fixed developer console warnings about views being created but not
   registered.][3181]
+- [Fixed "Invalid version" error during sending text change to Language
+  Server.][3186] The error happened during project opening and after new node
+  creation.
```

Comment:
On the first place, I would say that an error in the console is fixed, because the user will notice rather log in the console than the error in Language Server communication.

Reply:
❤️ thanks! done